Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -558,6 +558,27 @@ def test_no_ec_commands(command, args, response, expected_hr, expected_auto_extr
     assert expected_auto_extract == result[3]  # ignore_auto_extract is in the 4th place in the result of the command
 
 
+def test_delete_attachment_command(mocker):
+    client = Client('server_url', 'sc_server_url', 'cr_server_url', 'username', 'password',
+                    'verify', 'fetch_time', 'sysparm_query', 'sysparm_limit', 'timestamp_field',
+                    'ticket_type', 'get_attachments', 'incident_name')
+
+    mocker.patch.object(client, 'delete_attachment', return_value=None)
+    result = delete_attachment_command(client=client, args={"file_sys_id": "1234"})
+    assert 'Attachment with Sys ID 1234 was successfully deleted.' in result[0]
+
+
+def test_delete_attachment_command_failed(mocker):
+    client = Client('server_url', 'sc_server_url', 'cr_server_url', 'username', 'password',
+                    'verify', 'fetch_time', 'sysparm_query', 'sysparm_limit', 'timestamp_field',
+                    'ticket_type', 'get_attachments', 'incident_name')
+
+    mocker.patch.object(client, 'delete_attachment', return_value="Error")
+    with pytest.raises(DemistoException) as e:
+        delete_attachment_command(client=client, args={"file_sys_id": "1234"})
+    assert 'No Record found.' in str(e)
```

Comment:
This UT is failing in build.

Reply:
fixed. The error message has been changed